.github/workflows/bundle_with_dakota_macos.yml

```diff
@@ -190,8 +207,8 @@ jobs:
         pv=$(echo ${{ matrix.python-version }} | sed 's/\.//g')
         unfixed_wheel_path=$(find /tmp/carolina_dist_unfixed -name "carolina*$pv*whl")
         echo "Found Carolina wheel at $unfixed_wheel_path"
-        delocate-listdeps $unfixed_wheel_path
-        delocate-wheel -w /tmp/carolina_dist -v $unfixed_wheel_path 
+        DYLD_LIBRARY_PATH=${EXTENDED_DYLD_LIBRARY_PATH} delocate-listdeps $unfixed_wheel_path
+        DYLD_LIBRARY_PATH=${EXTENDED_DYLD_LIBRARY_PATH} delocate-wheel -w /tmp/carolina_dist -v $unfixed_wheel_path 
```

Comment:
Consider something like;
```diff
+         # Avoid MacOS security features prohibiting setting DYLD_LIBRARY_PATH
+         # Set DYLD_LIBRARY_PATH to be part of context where following commands are executed
```

Reply:
It is added above the lines here as shown 
``` 
        # Avoid MacOS security features prohibiting setting DYLD_LIBRARY_PATH
        # ref issue: https://github.com/pypa/cibuildwheel/issues/816
        EXTENDED_DYLD_LIBRARY_PATH="${libgccpath}:${HOME}/${INSTALL_DIR}/lib:$INSTALL_DIR/bin"
        DYLD_LIBRARY_PATH=${EXTENDED_DYLD_LIBRARY_PATH} delocate-listdeps $unfixed_wheel_path
        DYLD_LIBRARY_PATH=${EXTENDED_DYLD_LIBRARY_PATH} delocate-wheel -w /tmp/carolina_dist -v $unfixed_wheel_path 
```